components/script/dom/htmlimageelement.rs

```diff
@@ -849,8 +905,12 @@ impl HTMLImageElement {
                         // FIXME(nox): According to the spec, setting the current
                         // request to the broken state is done prior to queuing a
                         // task, why is this here?
+                        // Moving this outside the task results in intermittent failure of
+                        // non-active-document.html tests.
```

Comment:
Not sure that this if this helpful or just noise — would be open to removing it and the other similar comment.

Reply:
Please file an issue about this with any additional information you have. This sounds like something that's worth investigating.